Plugins/Bitbucket/BitbucketPullRequestForm.cs

```diff
@@ -24,7 +23,7 @@ public partial class BitbucketPullRequestForm : GitExtensionsFormBase
         private readonly Settings _settings;
         private readonly BindingList<BitbucketUser> _reviewers = new BindingList<BitbucketUser>();
 
-        public BitbucketPullRequestForm(BitbucketPlugin plugin, ISettingsSource settings, GitUIEventArgs gitUiCommands)
+        internal BitbucketPullRequestForm(Settings settings)
```

Comment:
Why has the visibility changed?

Reply:
Because `Settings` class is internal. Maybe I should have changed that class to public :haha:

Reply:
Changed it back.